gitbook/other/MIGRATING_TO_V8.0.0.MD

```diff
@@ -0,0 +1,222 @@
+# Migrating to V8.0.0
```

Comment:
Not sure this file should be here actually 🤔 It belongs to the general readme of the repo, not gitbook.